picard/formats/id3.py

```diff
@@ -405,6 +407,33 @@ def _save(self, filename, metadata):
             except:
                 pass
 
+    def _remove_deleted_tags(self, metadata, tags):
+        """Remove the tags from the file that were deleted in the UI"""
+        for tag in metadata.deleted_tags:
+            real_name = self._get_tag_name(tag)
+            if real_name == 'POPM':
+                for key, frame in tags.items():
+                    if frame.FrameID == 'POPM' and frame.email == config.setting['rating_user_email']:
+                        del tags[key]
+            elif real_name in tags:
+                del tags[real_name]
+
+    def supports_tag(self, name):
+        return name in self.__rtranslate or name in self.__rtranslate_freetext\
```

Comment:
Please use parenthesis instead of escaping end of line:
```python
return (name in self.__rtranslate
        or name in self.__rtranslate_freetext
...
```